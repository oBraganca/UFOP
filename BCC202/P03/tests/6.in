20
I Pedro
253441 pulse 8 2004 Alice 200
589444 hb20 4 1998 Arthur 203
956824 tcross 5 1984 Pedro 206
676717 nivus 4 2017 Davi 207
659791 mobi 7 1991 Helena 204
760863 gol 2 2023 Pedro 206
120155 nivus 4 1994 Helena 204
397020 creta 6 2002 Isabella 208
827471 mobi 5 1981 Pedro 206
339859 yars 9 2004 Alice 200
824271 fox 1 2016 Bernardo 205
834266 fox 3 2001 Alice 200
730823 gol 7 1999 Bernardo 205
790339 gol 2 1983 Miguel 201
757539 nivus 4 2008 Arthur 203
598145 tcross 2 1990 Alice 200
132310 creta 2 1985 Sophia 202
350667 tcross 5 2013 Helena 204
896485 tcross 8 1986 Bernardo 205
848114 argo 8 2018 Miguel 201
