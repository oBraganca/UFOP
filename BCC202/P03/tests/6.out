mobi [1981|5|827471]
gol [2023|2|760863]
tcross [1984|5|956824]
